x-pack/plugins/monitoring/server/lib/elasticsearch/shards/get_shard_stats.js

```diff
@@ -43,6 +43,7 @@ export function getShardStats(req, esIndexPattern, cluster, { includeNodes = fal
     index: esIndexPattern,
     size: 0,
     ignoreUnavailable: true,
+    rest_total_hits_as_int: true,
```

Comment:
In this case we're actually interested in the `hits.total` being non-zero, but we cannot rely on `hits.hits.length` because we use `size: 0`.

In this case though, we can just check the `indices.buckets` to see if it has a `length > 0`, then use it that way. Alternatively, we could just drop the parameter and rely on the by-default inaccurate count coming back to still be `>= 1` by changing it to `hits.total.value` from `hits.total`.

Reply:
Sounds good, will adjust